mkt/search/filters.py

```diff
@@ -204,6 +205,23 @@ class ExtensionSearchFormFilter(SearchFormFilter):
     VALID_FILTERS = ['author.raw']
 
 
+class HomescreenFilter(BaseFilterBackend):
+    """
+    A django-rest-framework filter backend that filters in or out homescreens.
+    """
+    def filter_queryset(self, request, queryset, view):
+        is_homescreen = request.GET.get('is_homescreen')
```

Comment:
ah, I see what you mean. fireplace is already using `is_homescreen` so let's fix it after next week